### PR TITLE
Chore: Support other pr titles

### PR DESCRIPTION
**Pull request by sampaiodiego**
## Proposed changes (including videos or screenshots)

We have at least these two additional automations we need to support:
- #22491 
- #22493

### CODE REVIEW

**Comment by KevLehman** [comment]
Can we add a custom start to the lingohub ones? 🙈 like `[lang] language update from...`. It would be cool.

**Comment by sampaiodiego** [comment]
> Can we add a custom start to the lingohub ones?  like `[lang] language update from...`. It would be cool.

yes, it is configurable on LingoHub end:
image

**Comment by KevLehman** [comment]
Then, for the future, maybe we can add `[lang]` as a supported PR title tag 😬 . The rest looks good.

File: .github/pr-title-checker-config.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	  "color": "B60205"
 	},
 	"CHECKS": {
-	  "regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Bump|Revert):) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+)",
+	  "regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Bump|Revert):) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Language update from LingoHub|Merge master into develop)",
```
Comment (KevLehman) [inline .github/pr-title-checker-config.json:7]:
```suggestion
	  "regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS|LANG)\\])?|(?:Regression|Chore|Bump|Revert|Lang):) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Language update from LingoHub|Merge master into develop)",
```

**Comment by sampaiodiego** [comment]
I agree.. not sure it should be a tag (and have its own category on release notes) or just a prefix like `i18n:` (something that goes into "minor changes") wdyt?

**Comment by KevLehman** [comment]
That's great 🥳 I agree with that

**Review by KevLehman — APPROVED** [review]

**Pull request merged**